### PR TITLE
Translate thanks page

### PR DESCRIPTION
**Pull request by PaNaVTEC**
Fixes https://github.com/codurance/site/issues/521

### CODE REVIEW

File: _i18n/es.yml [hunk 1]
```diff
@@ -341,3 +341,10 @@ latest-publications:
   posts: Últimos posts
 author:
   publications-title: Publicaciones
+thanks:
+  get-back-soon: Agradecemos que se haya tomado el tiempo de escribirnos. Nos pondremos en contacto contigo pronto.
```
Comment (guillemf) [inline _i18n/es.yml]: You use third person and second in the two sentences. It should be one or the other but not mix both. Ex
Agradecemos que te hayas tomado el tiempo de escribirnos. Nos pondremos en contacto contigo pronto.

or

Agradecemos que se haya tomado el tiempo de escribirnos. Nos pondremos en contacto con usted pronto.

File: _i18n/es.yml [hunk 2]
```diff
@@ -341,3 +341,10 @@ latest-publications:
...
+  our-blog: Nuestros posts
+  video-section: Sección de videos
+  services-we-offer: Servicios que ofrecemos
+  until-then: >
+    Hasta entonces, puedes consultar algunos de nuestros recursos:
```
Comment (guillemf) [inline _i18n/es.yml:350]: Same as above.
Hasta entonces, puedes consultar algunos de nuestros recursos:

or 

Hasta entonces puede consultar alguno de nuestros recursos:

Based on the previous choice.

File: _i18n/es.yml (hunk 1 above)
Comment (alvarogarcia7) [reply _i18n/es.yml]: Agree: pick one style or the other.

I prefer more direct (tu) in headlines and call to action and more formal (usted) in longer texts

**Review by alvarogarcia7 — CHANGES_REQUESTED** [review]

**Comment by PaNaVTEC** [comment]
Changed to use 'tu' instead. Let me know if there is anything else that we can improve. Thanks @guillemf @alvarogarcia7 @ananogal

File: _i18n/es.yml (hunk 2 above)
Comment (CarlosMChica) [reply _i18n/es.yml:350]: "Nos pondremos en contacto lo antes posible" instead of "nos pondremos en contacto contigo pronto".

Sounds better I'd say

**Review by guillemf — APPROVED** [review]

File: _i18n/es.yml (hunk 2 above)
Comment (PaNaVTEC) [reply _i18n/es.yml:350]: It sounds better for me "Nos pondremos en contacto contigo pronto" becase we specify to who we are going to contact (contigo).
"Nos pondremos en contacto contigo lo antes posible" works for me as well.
Any other opinion on that ? @alvarogarcia7 @guillemf

Comment (alvarogarcia7) [reply _i18n/es.yml:350]: @PaNaVTEC looks good to me

**Review by alvarogarcia7 — APPROVED** [review]

**Pull request merged**